torch/csrc/jit/mobile/model_compatibility.cpp

```diff
@@ -220,6 +299,42 @@ ModelCompatCheckResult is_compatible(
     result.errors.emplace_back(s.str());
   }
 
+  SupportedType supported_type = runtime_info.supported_types;
```

Comment:
Consider refactoring this long function to several functions like typeCompactability(), operatorCompactability(), etc.

Reply:
Yeah let me have a separate pr for it